addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -163,6 +163,10 @@ <H2>Strict Transport Security Header Scanner</H2>
 
 Redirects to HTTPS URLs on the same domain will only be reported at Low threshold.
 
+<H2>Sub Resource Integrity Attribute Missing</H2>
```

Comment:
Should mention the trusted domains config.

Reply:
Should we do a reference to **Reverse Tabnabbing**?
Like `rules.domains.trusted` can be used. Look at Reverse Tabnabbing for more information`?

Reply:
How about extracting a section with that, that both can refer to? That's probably more noticeable when promoting the rules (if one is promoted before the other).